loadtest/control/simulcontroller/websocket.go

```diff
@@ -36,6 +36,12 @@ func (c *SimulController) wsEventHandler(wg *sync.WaitGroup) {
 				break
 			}
 
+			if ack, ok := ev.GetData()["should_ack"]; ok && ack.(bool) {
```

Comment:
Double-checking: What should we do if there's no `"should_ack"` key in the event data? Just skip?

Reply:
Correct. we only want to actually send the ACK back if the posted event has this flag, if not we shouldn't do anything.

Reply:
Got it!